### DISCUSSION TITLE
Test Notification

### DISCUSSION THREAD

**Discussion by munoztd0** [munoztd0/DAP_2021#2] [Announcements]
This is just to test that you are receiving notifications from this discussion feed. @lucilefavero  @ap1987-ap  @faureunige  @danielsgranja @dmt411 
So if you see this, could you just answer anything or react to it, thanks.

Moreover, some of you haven't sent their data with a minor description and the main question, but no worries I will let you until Thursday afternoon (4pm) to send it to me.

**Comment by ap1987-ap** [comment]
Hi, i got it. And i will send you the dataset and the description in the next hour.

Thanks and Best
Anja

**Comment by danielsgranja** [comment]
I got it too!

**Comment by faureunige** [comment]
Got it too, will publish my dataset as soon as possible thanks !

**Comment by lucilefavero** [comment]
Hi, I got it, thanks .
